les/vflux/client/serverpool_test.go

```diff
@@ -54,6 +54,8 @@ type ServerPoolTest struct {
 	db                   ethdb.KeyValueStore
 	clock                *mclock.Simulated
 	quit                 chan chan struct{}
+	stopping             bool
+	queryWg              *sync.WaitGroup
```

Comment:
Why not just use a `sync.WaitGroup` instead of a pointer?

Reply:
Updated to remove the pointer

Reply:
Changed it back and added comments to explain why it needs to be recreated each time and how the waitgroup is used safely :)